### ISSUE TITLE
Take pinned repositories into account

### ISSUE THREAD

**Issue by saxbophone** [resume/resume.github.com#276]
Github allows users to "pin" up to 6 repositories they own to their Github profile page.

It could be useful to take this into account when deciding which repositories to highlight, as an alternative to the "popular repositories" section.

**Comment by Elsie19** [comment]
Yeah, for instance if you contribute to a popular program regularly that happens to be in an organization.

**Comment by saxbophone** [comment]
> Yeah, for instance if you contribute to a popular program regularly that happens to be in an organization.

That's a good point too, I didn't think of that —I was thinking of user profile pages, but I agree, organisation profile pages should count too. Maybe there should be a separate option to enable both...
